client/src/components/QandA/Answer.jsx

```diff
@@ -15,20 +15,30 @@ function Answer({ answer, helpfulHandler, reportHandler }) {
   return (
     <div className="answerContainer">
       <div className="answerBody">
-        A:
-        {' '}
         {answer.body}
+        {answer.photos.length
+          ? (
+            <div className="photos">
+              {answer.photos.map((photo) => (
+                <img className="answer-photo" src={photo.url} alt="" />
```

Comment:
Images should have an default image src if the image url does not work.
For example, for my product cards I defined the following: 
` const defaultImgURL = 'https://www.freeiconspng.com/uploads/no-image-icon-15.png';`
'defaultImgURL' is just an 'Image not available' image
So your img tag should be this: `<img className="answer-photo" src={photo.url || defaultImgURL} alt="" />`